feature-pack-build-maven-plugin/src/main/java/org/wildfly/build/plugin/FeaturePackBuildMojo.java

```diff
@@ -107,7 +107,7 @@
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
         copyContents();
-        try (FileInputStream configStream = new FileInputStream(new File(configDir, configFile))) {
+        try (FileInputStream configStream = new FileInputStream(new File(project.getBasedir(), configFile))) {
```

Comment:
Ah, you are right! For some reason I thought `File(File parent, String child)` was doing basically the same thing as `Path.resolve(Path)`.